.github/workflows/tests.yml

```diff
@@ -67,5 +67,41 @@ jobs:
                 run: composer require typo3/minimal "^${{ matrix.typo3-versions }}" --prefer-dist --no-progress --no-suggest
 
             -
-                name: Automated Testing
-                run: composer test
+                name: Automated Unit Testing
+                run: composer test:unit
+
+    integrationtest:
+        runs-on: ubuntu-latest
+
+        strategy:
+            max-parallel: 2
+            matrix:
+                php-versions: [7.4, 7.3, 7.2]
+                typo3-versions: [9.5]
+
+        name: PHP ${{ matrix.php-versions }} with TYPO3 ${{ matrix.typo3-versions }}
```

Comment:
I recommend adding the test type in the name for a hint of the type in the list view.

![image](https://user-images.githubusercontent.com/26324456/95840136-166b4c80-0d44-11eb-8770-c91d683f04c0.png)


```suggestion
        name: Integration Test with PHP ${{ matrix.php-versions }} and TYPO3 ${{ matrix.typo3-versions }}
```

and 
```
name: Unit Test with PHP ${{ matrix.php-versions }} and TYPO3 ${{ matrix.typo3-versions }}
```

If the names are to long to show in list you could trim i a bit to 
`Integration PHP 7.4; TYPO3 10.4`